docs/explanation/gno-tooling/cli/common-params.md

```diff
@@ -0,0 +1,16 @@
+---
+id: gno-tooling-common-params
+---
+
+# Common Parameters
+
+Below is a list of common parameters.
```

Comment:
I'm unsure about the file's usefulness. Shouldn't we keep it as a section within a binary document instead?

Reply:
Dropped as part of:

https://github.com/gnolang/gno/pull/1046#discussion_r1386610796